pkg/minishift/openshift/service.go

```diff
@@ -148,7 +173,7 @@ func getServiceURLsOutput(namespace string) (string, error) {
 	return byteArrayToString(cmdOut), nil
```

Comment:
hmm, I am not sure about the value of _byteArrayToString_. I think it is confusing for a Go programmer, after all it is just _string(data)_